styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -1,3 +1,218 @@
+// Customer Stories Feature Colors
+$cust-stories-red: #9d0000;
+$cust-stories-teal: #00b4c3;
+$cust-stories-purple: #563777;
+$default-bg-img: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/IMACbanners/collaboration-2.jpg';
```

Comment:
So this is just the default if no image is selected. The user should have the option to override this by choosing the image in Drupal.